src/app/payment/paymentMethod/MasterpassPaymentMethod.tsx

```diff
@@ -20,8 +20,17 @@ const MasterpassPaymentMethod: FunctionComponent<MasterpassPaymentMethodProps &
         },
     }), [initializePayment]);
 
+    const formatLocale = useCallback((localeLanguage: string) => {
+        const auxLocale = localeLanguage.replace(/[-_]/g, '_');
```

Comment:
Why are we replacing `_` with `_`?

```suggestion
        const auxLocale = localeLanguage.replace(/[-]/g, '_');
```